test/integration/required-server-files-ssr-404/test/index.test.js

```diff
@@ -236,7 +236,7 @@ describe('Required Server Files', () => {
           {
             headers: {
               'x-matched-path': '/fallback/first',
-              'x-now-route-matches': '1=first',
+              'x-now-route-matches': 'nxtPslug=first',
```

Comment:
So we already had test showing symptoms of the bug?

Reply:
This is specific to the way that Vercel handles the matches. We used to previously support both the index based and name based matches, but this isn't supported on Vercel anymore, which is what I suspect cause this to surface.